1.0/gradle/vividus-library.gradle

```diff
@@ -70,6 +70,8 @@ def toolchainPropertyName = 'javaToolchainEnabled';
 
 def useToolchain = project.hasProperty(toolchainPropertyName) ? project.getProperty(toolchainPropertyName).toBoolean() : false
 
+ext.defaultJvmArgs = '--add-exports=java.desktop/sun.awt.image=ALL-UNNAMED';
```

Comment:
```suggestion
ext.defaultJvmArgs = ['--add-exports=java.desktop/sun.awt.image=ALL-UNNAMED'];
```

Reply:
but if we do it will be simpler to add it using the list

Reply:
the chance is too low, additionally even if you pass '--add-ex... --second-arg' it will still work

Reply:
It might work for now, but you can't rely on that, if API accepts the list, then it should be a list, not a single-entry list with the merged attributes. 
And secondly, it is not an extra collection, you are still creating it here:
  ```
jvmArgs += [ defaultJvmArgs ]
```

Reply:
Fixed the extra collection, for I would still prefer to use plain string instead of creating redundant list and avoid additional list concatenation, but once we have one more argument (too low chance for this) I'll definitely use list

Reply:
Fixed the extra collection, for I would still prefer to use plain string instead of creating redundant list and avoid additional list concatenation, but once we have one more argument (too low chance for this) I'll definitely use list